crates/compiler/can/tests/test_can.rs

```diff
@@ -382,7 +382,7 @@ mod test_can {
         let arena = Bump::new();
         let CanExprOut { problems, .. } = can_expr_with(&arena, test_home(), src);
 
-        assert_eq!(problems.len(), 0);
+        assert_eq!(problems, Vec::new());
```

Comment:
It actually prefer it as-is, because this way if the assertion fails, the failure will print out what was inside the vec that we expected to be empty! 😄

Reply:
Yep; that's exactly why I made this change! (this assert failed, and I wanted to see what problem was there)